bats/tests/helpers/vm.bash

```diff
@@ -115,7 +115,7 @@ start_container_engine() {
             add_profile_bool "WSL.integrations.${WSL_DISTRO_NAME//./$RD_PROTECTED_DOT}" true
         fi
         add_profile_bool containerEngine.allowedImages.enabled "$image_allow_list"
-        add_profile_list containerEngine.allowedImages.patterns "$registry"
+        #add_profile_list containerEngine.allowedImages.patterns "$registry"
```

Comment:
Because `$registry` overrides the list of allowed images that are added to the locked profile before we run `start_container_engine`. This is just a temporary workaround for me to test if the tests are working on all platforms. But I'm planning to fix this before merging it